game-of-life/src/App.js

```diff
@@ -84,13 +83,13 @@ function App() {
 	}; //closing function alive cells
 
 	const updateBoard = () => {
-		const clonedBoard = [...boardStatus];
+		let clonedBoard = [...boardStatus];
```

Comment:
nice way of using spread operator to not mutate the state of your board!

Reply:
unfortunately this doesn't work with multidimensional arrays so I had to hack a little bit once I realized it by doing let clonedBoard = JSON.parse(JSON.stringify(boardStatus));`